examples/css3d_molecules.html

```diff
@@ -370,11 +370,7 @@
 
 						//
 
-						bond = document.createElement( 'div' );
-						bond.className = 'bond';
-						bond.style.height = bondLength + 'px';
-
-						const joint = new THREE.Object3D( bond );
+						const joint = new THREE.Object3D();
```

Comment:
> As far as I can tell, passing an HTML element in an Object3D constructor doesn't do anything.

Good catch! However, `bond` is later used in the code to create another instance of `CSS3DObject`. And it's not right that multiple objects share the same DOM element. So please restore the bond creation and just remove the parameter from the ctor call of `Object3D`.

Reply:
🤦 Indeed. I moved the creation of the new `bond` element closer to the usage if that's okay.